### PR TITLE
Story/demo 3

### PR DESCRIPTION
**Pull request by ramo**
- [x]  one
- [x]  two 
- [x]  three

### CODE REVIEW

**Comment by ramo** [comment]
this is a test comment.

File: file1.txt [hunk 1]
```diff
@@ -1,9 +1 @@
-This is a test file.
-ramo-test have added something.
-
-
-
-
-
-I have some more items
-ramo-test added something for story/demo-2
```
Comment (ramo-test) [inline file1.txt:9]: Do the requested change as per our discussion.

**Review by ramo-test — CHANGES_REQUESTED** [review]

File: file1.txt [hunk 2]
```diff
@@ -1,9 +1 @@
...
+This is a test file.
```
Comment (ramo-test) [inline file1.txt:1]: this is again a test.

**Review by ramo-test — CHANGES_REQUESTED** [review]

File: file1.txt [hunk 3]
```diff
@@ -1,9 +1 @@
-This is a test file.
-ramo-test have added something.
-
```
Comment (ramo-test) [inline file1.txt:3]: add something.

**Review by ramo-test — CHANGES_REQUESTED** [review]

File: file1.txt (hunk 2 above)
Comment (ramo) [reply file1.txt:1]: test

**Comment by ramo-test** [comment]
testing a comment.

**Comment by ramo-test** [comment]
test

File: file1.txt (hunk 2 above)
Comment (ramo-test) [reply file1.txt:1]: test

Comment (ramo) [reply file1.txt:1]: test

**Comment by ramo-test** [comment]
test

File: file1.txt [hunk 4]
```diff
@@ -1,9 +1 @@
-This is a test file.
```
Comment (ramo-digisight) [inline file1.txt:1]: What is this?

**Review by ramo-digisight — CHANGES_REQUESTED** [review]

**Pull request merged**